jekyll/_cci2/pipelines.adoc

```diff
@@ -12,48 +11,63 @@ contentTags:
 :page-layout: classic-docs
 :page-liquid:
 
-CircleCI pipelines are the highest-level unit of work, and they are described by a configuration file, for example, `.circleci/config.yml`. _Triggers_ can be set up to automate when a pipeline will be triggered. Pipelines coordinate the execution of workflows and jobs to build, test, and deploy your code. Workflows can be conditional, based on xref:workflows#using-filters-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters].
 
-Pipelines include your workflows, which coordinate your jobs. Pipelines have a fixed, linear lifecycle, and are associated with a specific actor. Pipelines trigger when a change is pushed to a project that has a CircleCI configuration file included, and can also be scheduled, triggered manually through the CircleCI app, or using the API.
+CircleCI pipelines are the highest-level unit of work in your CI/CD process. Defined by a configuration file (typically `.circleci/config.yml`), pipelines orchestrate the execution of _workflows_ which run _jobs_ to build, test, and deploy your code.
 
-== Dashboard
-When visiting your CircleCI dashboard, you are shown a list of recently-triggered pipelines for your organization.
+Pipelines can be triggered in the following ways:
 
-image::/docs/assets/img/docs/pipelines-dashboard.png[Screenshot of the pipelines dashboard in the CircleCI app]
+* Automatically: code changes, scheduled events, custom webhooks
+* Manually: CircleCI app or API
 
-[#pipeline-architecture]
-== Pipeline architecture
+Pipelines provide a flexible framework for creating conditional workflows based on xref:workflows#using-filters-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters], allowing you to customize your CI/CD process to fit your project's needs.
 
-A pipeline is composed of workflows, which are composed of jobs. By navigating from a pipeline to a specific job, you can access your job output, test results, and artifacts through several tabs.
+== Pipelines dashboard
+Your pipelines dashboard displays recently-triggered pipelines for your organization. Use the various filters to find the pipelines you need. Click through to view workflow maps and individual job steps. Access quick controls to rerun and cancel workflows
```

Comment:
Would it make sense to link from "view workflow maps and individual job steps" to docs/jobs-steps/#access-jobs-and-steps-in-the-circleci-app ?